src/main/resources/l10n/JabRef_de.properties

```diff
@@ -1223,9 +1223,9 @@ Stored_entry=Eintrag_gespeichert
 
 Strings=Ersetzen
 
-Strings_for_database=Strings_für_die_Datei
+Strings_for_database=Strings_für_die_Datenbank
 
-Subdatabase_from_AUX=Teildatenbank_aus_AUX-Datei
+Subdatabase_from_AUX=Teildatenbank_aus_AUX-Datenbank
```

Comment:
I thought because a subdatabase was created from AUX, AUX had to be a database. I will revert it.